content/wiki/Coaching-a-team.md

```diff
@@ -0,0 +1,91 @@
+the type of problems appearing in contests. There are multiple books published that can help you to do this. A small 
+incomplete selection:
+* [Competitive Programming 4th Edition](https://cpbook.net/) ([2nd edition downloadable for free](https://cpbook.net/details?cp=2))
+* [Programming Challenges: The Programming Contest Training Manual](https://books.google.nl/books/about/Programming_Challenges.html)
+* [Guide to Competitive Programming: Learning and Improving Algorithms Through Contests](https://books.google.nl/books?id=3JbiDwAAQBAJ)
+
+Another option is to practice previous problems. This can be done in [open Kattis](https://open.kattis.com/) or using
+the [CHipCie problem archive](https://commissies.ch.tudelft.nl/chipcie/archive). Having regular sessions where teams
+practice in a simulated contest environment. Having a time with one computer available, factors on roles of team members
+can be fine-tuned. Think about setting up the code environment, selecting the first problems and efficiently using the 
+computer. 
+
+# Preparing for a contest
+A coach can help a team prepare for a contest. Most contest allow for a Team Reference Document (TRD, sometimes called 
+Team Contest Reference (TCR)), sometimes may bring a keyboard and/or mouse and a dictionary. This information can
+usually be found in the rules and systems pages on the website. If you can't find this information, don't hesitate
+to contact to organisation to ask.
+
+For TRD, a selection of following templates exist which can be used for a team as basis:
+* [TU Delft template](https://commissies.ch.tudelft.nl/chipcie/resources/tcr-template.zip) ([pdf](https://commissies.ch.tudelft.nl/chipcie/resources/tcr.pdf))
+* [Does god play Tetris](https://github.com/alexjbest/icpc-tcr/blob/master/teamref.pdf)
+* [Team hacKIT](https://github.com/niklasb/tcr/blob/master/tcr.pdf)
+* [Omogen Heap](https://github.com/kth-competitive-programming/kactl/blob/main/kactl.pdf)
+
+The TRD should be in a folder, or at least be stapled together. Make sure the teams bring a printed version to the 
+contest, since not all contests offer the opportunity to print the document on site.
+
+Other tips you can give your teams as coach is to check the schedule and see when/if there is dinner. If the award
+ceremony is before the dinner, advice your teams to bring some snacks for after the contest. 
+
+Another common issue is the temperature in the contest halls. It can heat up very fast with many people and pc's at the 
+same time. Dressing in easily removable layers can help manage the temperature. Note that the T-shirt should usually
+be worn on top and make sure your contestants are aware of it.
+
+In case one of your team members has dietary restrictions (vegetarian, vegan, gluten free, nut allergy), make sure this i
+s known by the organisation. In ICPC there is a special needs field the contestants should fill in. It depends on per 
```

Comment:
```suggestion
In case one of your team members has dietary restrictions (vegetarian, vegan, gluten free, nut allergy), make sure this i
s known by the organisation. In ICPC there is a special needs field the contestants should fill in. It depends per 
```